common/domain/handler.go

```diff
@@ -258,6 +258,7 @@ func (d *HandlerImpl) RegisterDomain(
 			domainRequest.ReplicationConfig,
 			domainRequest.ConfigVersion,
 			domainRequest.FailoverVersion,
+			persistence.InitialPreviousFailoverVersion,
```

Comment:
If the constant will be used by the application layer logic then I think it's probably better to define it in the common package, just like `EmptyVersion`.

Reply:
Updated